src/Analyzers/CSharp/CodeFixes/MakeLocalFunctionStatic/MakeLocalFunctionStaticCodeFixHelper.cs

```diff
@@ -134,13 +133,18 @@ internal static class MakeLocalFunctionStaticCodeFixHelper
                         {
                             syntaxEditor.ReplaceNode(
                                 identifierNode,
-                                (node, generator) => generator.IdentifierName(parameter.Name.ToIdentifierToken()).WithTriviaFrom(node));
+                                (node, generator) => SyntaxFactory.IdentifierName(parameter.Name.ToIdentifierToken()).WithTriviaFrom(node));
```

Comment:
this existing SyntaxGenerator helper is internal.  Easiest workaround was to just do what it does (since we're at the C# layer).